doc/.gitignore

```diff
@@ -0,0 +1 @@
+/man.md
```

Comment:
apparently there's a missing line break which is a good practice to have

Reply:
Good catch, I usually rely on `pre-commit` to catch these things, but I forgot this one does not have it implemented yet.

Reply:
Oh, thanks for reminding, I completely forgot :D

Reply:
Ok, that's fixed now. Testing-farm is failing, but that's another can of worms to address in #160